src/app/frontend/deploy/deployfromsettings.html

```diff
@@ -81,37 +86,38 @@
 
 <kd-help-section>
   <md-checkbox ng-model="ctrl.isExternal" class="md-primary"
-             ng-model-options="{ debounce: { 'default': 500, 'blur': 0 } }">
+               ng-model-options="{ debounce: { 'default': 500, 'blur': 0 } }">
     Expose service externally
   </md-checkbox>
 </kd-help-section>
 
+<!-- advanced options -->
 <div ng-show="ctrl.isMoreOptionsEnabled()">
   <kd-help-section>
-    <md-input-container>
+    <md-input-container class="md-block kd-deploy-block">
```

Comment:
don't we want to use this style potentially on another page? If a ui designer wishes to distinguish between pages, it would still be possible with something like "deploypage .kd-input-block"